unfoldingWord/translationCore
# Added local import dialog and USFM tests

#### This pull request addresses:
This PR fixes some workflow issues with USFM for better testing,
adds more testing for USFM to ensure handling of missing / out of sequence content,
lastly adds a dialog to let the user know the local import succeeeds

#### How to test this pull request:
Try importing a project that is USFM and ensure everything work as expected
Try running <code>npm run</code>


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/2586)
